### DISCUSSION TITLE
Measurement Units Selection

### DISCUSSION THREAD

**Discussion by djgeo** [opengisch/QField#3165] [Ideas]
I am a very satisfied user of QField. I have found no other app that meets my needs so well as QField. I do have one request that would expand the usefulness of Qfield. It would be nice if we would have an option for choosing different units of measurements for the measure tool. For length,  feet, km, miles would all be useful for different applications. And for area, square feet, acres, hectares, square miles would be very useful as well.

**Comment by 711wolf** [comment]
It would also be extremely useful to be able to select 'feet' as a unit for reporting elevation, positional accuracy, etc. I know most of the world is metric, but the US uses feet, and I work in the US, so it would be very valuable to have the option to select map and elevation units rather than only having meters. In addition, it would be great if QField could match the reported units to the units of the projection (perhaps this could be a toggle-able option) so that if I'm working in a project whose projection units are feet, QField could be set to automatically report all elevation and length values in feet, and area values in square feet. QField is wonderful and these simple unit conversion and detection features would make it even more amazing! Thank you.

**Comment by nirvn** [comment]
@djgeo , QField does support those units, e.g. here's feet measurement:

![image](https://github.com/opengisch/QField/assets/1728657/a499986f-7444-4d70-82bd-ac1b050b8431)

This is set in your project properties within QGIS:

![image](https://github.com/opengisch/QField/assets/1728657/1a6bfae2-8ebf-48ca-af89-9efaa202d918)

**Reply by djgeo** [reply]
@nirvn

Distance measurement unit settings are applied in Qfield.
But area measurements units stay at hectares in Qfield
![Screenshot_20230826_090954](https://github.com/opengisch/QField/assets/83652907/48120807-4d4e-46a5-9fb1-09f3c5531477)
![Screenshot 2023-08-26 091153](https://github.com/opengisch/QField/assets/83652907/fbb0b0c6-be0a-4c11-84cb-731c13ec0150)

**Reply by nirvn** [reply]
@djgeo , can you send a sample project? I am not able to reproduce this over here.

**Reply by nirvn** [reply]
@djgeo , ignore that, the issue has been found and fixed: https://github.com/opengisch/QField/pull/4538 -- thanks for reporting.
